src/website/app/demos/Tooltip/examples/placement.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import Button from '../../../../../Button';
+import { createStyledComponent } from '../../../../../styles';
+import Tooltip from '../../../../../Tooltip';
+import IconDelete from 'mineral-ui-icons/IconDelete';
+
+const DemoLayout = createStyledComponent('div', {
+  height: '200px',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center'
+});
+
+const content = 'Change the `placement` prop to any of the values noted below';
+
+export default {
+  id: 'placement',
+  title: 'Placement',
+  description: `The \`placement\` prop determines the initial placement of the Tooltip content relative to the trigger.
+The Tooltip will still react to viewport edges and scrolling.`,
+  scope: { Button, content, DemoLayout, IconDelete, Tooltip },
+  source: `
+    /*
```

Comment:
I don't think there is a need to enumerate these placement values.  The values are available in the props table and are also pretty intuitive.  This also makes the example inconsistent with other placement examples across the site.  We want to try to make them seem like they were all developed with the same intent.  If a change is needed for one, it should ideally be made to all.

Reply:
I'd suggest to default the placement in this example to `bottom`.  The `left-start` is a little strange, but is useful if the user is changing values.

Reply:
I'd suggest to not use a `minimal` icon button for the trigger in this example as the placement of the tooltip appears farther away from the trigger than necessary.  This is due to it being a controlled tooltip and the button not being hovered/focused and thus not "showing" its actual size.

Reply:
The "left-start" initial placement is also negatively affecting the page layout on mobile.

<img width="150" alt="screen shot" src="https://user-images.githubusercontent.com/202773/36031800-35ed0216-0d69-11e8-9e85-8a72c7511fd6.jpg">
<img width="150" alt="screen shot" src="https://user-images.githubusercontent.com/202773/36031807-3acda8ee-0d69-11e8-9b35-9927c9afda3c.jpg">

Reply:
Addressed all of the above.